### ISSUE TITLE
The calculation principle of mask

### ISSUE THREAD

**Issue by 342600694** [AlvinYH/Faster-VoxelPose#53]
Hi, this is good work and I'm very inspired by it, but when I read the project_individual section of the code, I'm a little confused about the mask handling. The code is as follows:
        mask = ((1 - proposal_centers[:, 5:7]) / 2 * (self.voxels_per_axis[0:2] - 1)).int()
        mask[mask < 0] = 0
        # the vertical length of the bounding box is kept fixed as 2000mm
        mask = torch.cat([mask, torch.zeros((num_people, 1), device=device, dtype=torch.int32)], dim=1)

        # compute the valid range to filter the outsider
        start = torch.where(centers_tl + mask >= 0, centers_tl + mask, torch.zeros_like(centers_tl))
        end = torch.where(centers_tl + self.voxels_per_axis - mask <= self.fine_voxels_per_axis, centers_tl + self.voxels_per_axis - mask, self.fine_voxels_per_axis)
I want to know how the mask calculation works here.
